jf_agent/main.py

```diff
@@ -552,7 +568,8 @@ def download_data(config, creds, endpoint_jira_info, endpoint_git_instances_info
                 logger,
                 logging.INFO,
                 f'Obtained {git_config.git_provider}:{git_config.git_instance_slug} configuration, attempting download '
-                + f'in parallel with {config.git_max_concurrent} workers'
+                + f'in parallel with {config.git_max_concurrent} workers, '
+                  f'for {len(git_config.git_include_projects)} projects'
```

Comment:
missing a `+` here? hard to tell

Reply:
Don't think you need it right? 
```
In [1] print(f"test "
                              ...: "something")
test something
```
but added here: https://github.com/Jellyfish-AI/jf_agent/pull/256/commits/15c532dc895a33d7a4d6d1a4bad67b3fa708ab63

Reply:
Oh I guess I'm not actually sure. I'd double check that it works in a python shell but I'm sure you already did 👍